libgnucash/backend/xml/test/gtest-load-save-files.cpp

```diff
@@ -319,3 +320,68 @@ std::vector<std::string> ListTestCases ()
     EXPECT_FALSE(files.empty()) << "no files found in " << location;
     return files;
 }
+
+TEST_F(LoadSaveFiles, LoadAndVerifyKVP)
+{
+    auto location{g_getenv ("GNC_TEST_FILES")};
+    if (!location)
+        location = "test-files/load-save";
+
+    std::shared_ptr<QofSession> session{qof_session_new(qof_book_new()), qof_session_destroy};
+    ASSERT_NE(session.get(), nullptr);
+
+    auto filename{"xml-datafile.gnucash"};
```

Comment:
This needs a better name reflecting what its purpose is because all of the files are "XML data files" so this is too generic.